ark/jsonschema/string.ts

```diff
@@ -0,0 +1,26 @@
+import { rootSchema, type Intersection } from "@ark/schema"
+import type { Out, Type } from "arktype"
+import { JsonSchemaScope, type StringSchema } from "./scope.ts"
+
+export const validateJsonSchemaString: Type<
```

Comment:
I would rename all these "validate" functions to parse

Reply:
Done within a4defa88